scripts/install

```diff
@@ -72,14 +24,6 @@ then
 	echo "$bootstrap_peers" | grep -E '[0-9a-f]{64}@((\b25[0-5]|\b2[0-4][0-9]|\b[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}|([a-zA-Z0-9][a-zA-Z0-9-]{0,61}[a-zA-Z0-9]\.)+[a-zA-Z]{2,}):[0-9]{1,4}' || ynh_die --message="friend server id must have id with the following form : 1799bccfd7411eddcf9ebd316bc1f5287ad12a68094e1c6ac6abde7e6feae1ec@192.168.1.1:1234 or  1799bccfd7411eddcf9ebd316bc1f5287ad12a68094e1c6ac6abde7e6feae1ec@example.tld:1234"
 fi
 
-if [ "$datadir" = "/home/yunohost.app/__APP_NAME__/data" ]
-then
-	datadir="/home/yunohost.app/$app/data"
-fi
-
-# Register (book) web path
-ynh_webpath_register --app=$app --domain=$domain --path_url="/"
-
 #=================================================
 # LOOKING FOR VIRTUALISATION
```

Comment:
As explained in the PR 12, I'm confused about this check for virtualisation stuff ... `which modprobe` is super ambiguous as to what it's trying to check ... if we're trying to identify that we are in a container or a VM, I would strongly encourage to use `systemd-detect-virt` which is way more reliable

Ideally, the corresponding conditional apt dependencies should be handled in the manifest too using `packages_from_raw_bash`, cf for example my_webapp, etherpad_mypads or hubzilla

Reply:
should be done